Missing `override`

```bash
In file included from /Users/slauae/projects/libmesh/libmesh-clang/include/libmesh/parallel_mesh.h:23:
/Users/slauae/projects/libmesh/libmesh-clang/include/libmesh/distributed_mesh.h:278:16: warning: 'own_node' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
  virtual void own_node (Node & n);
               ^
/Users/slauae/projects/libmesh/libmesh-clang/include/libmesh/mesh_base.h:626:16: note: overridden virtual function is here
  virtual void own_node (Node &) {}
               ^
1 warning generated.
In file included from /Users/slauae/projects/moose/framework/build/unity_src/userobject_Unity.C:29:
In file included from /Users/slauae/projects/moose/framework/src/userobject/SolutionUserObject.C:24:
In file included from /Users/slauae/projects/libmesh/libmesh-clang/include/libmesh/parallel_mesh.h:23:
/Users/slauae/projects/libmesh/libmesh-clang/include/libmesh/distributed_mesh.h:278:16: warning: 'own_node' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
  virtual void own_node (Node & n);
               ^
/Users/slauae/projects/libmesh/libmesh-clang/include/libmesh/mesh_base.h:626:16: note: overridden virtual function is here
  virtual void own_node (Node &) {}
               ^
1 warning generated.
In file included from /Users/slauae/projects/moose/framework/build/unity_src/outputs_Unity.C:20:
In file included from /Users/slauae/projects/moose/framework/src/outputs/OversampleOutput.C:17:
/Users/slauae/projects/libmesh/libmesh-clang/include/libmesh/distributed_mesh.h:278:16: warning: 'own_node' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
  virtual void own_node (Node & n);
               ^
/Users/slauae/projects/libmesh/libmesh-clang/include/libmesh/mesh_base.h:626:16: note: overridden virtual function is here
  virtual void own_node (Node &) {}
               ^
1 warning generated.
```

--- a/include/mesh/distributed_mesh.h
+++ b/include/mesh/distributed_mesh.h
@@ -275,7 +275,7 @@ public:
    * well as changing n.id() and moving it in the mesh's internal
    * container to give it a new authoritative id.
    */
-  virtual void own_node (Node & n);
+  virtual void own_node (Node & n) libmesh_override;
 
   virtual void delete_node (Node * n) libmesh_override;
   virtual void renumber_node (dof_id_type old_id, dof_id_type new_id) libmesh_override;

